examples/asr/notebooks/1_ASR_tutorial_using_NeMo.ipynb

```diff
@@ -594,8 +592,8 @@
     "    # The print_func defines what gets printed.\n",
     "    print_func=partial(\n",
     "        monitor_asr_train_progress,\n",
-    "        labels=labels,\n",
-    "        logger=logger)\n",
+    "        labels=labels),\n",
+    "    tb_writer=neural_factory.tb_writer\n",
```

Comment:
I don't think you need to log this to tensorboard. Or if you want to log to tensorboard, maybe mention in the tutorial that we auto log to tensorboard and you can add a section on how to load the tensorboard file.

Reply:
Sure, Added section on how to load tensorboad file. Also as torch doesn't install tensorboard added that to our requirements.txt file

Reply:
This should be discussed later as in some files (for eg: examples/image/resnet50.py) we are using tensorboardX. May be we should standardize this.